test_projects/expected_state/Makefile

```diff
@@ -0,0 +1,148 @@
+.PHONY: \
+	dev \
+	test \
+	compile \
+	seed-db \
+	reset-db \
+	reseed-db \
+
+	set-remotes \
+	heroku-console \
+	heroku-capture-backup \
+	heroku-latest.dump \
+	import-latest-db \
+	heroku-ci-deploy \
+
+
+	devstack \
+	devstack-build \
+	devstack-clean \
+	devstack-shell \
+	devstack-run
+
+
+DEFAULT_GOAL: help
+
+# TODO read database name from Mix.Config
+PGDATABASE?=skafolder_tester_backend_dev
+
+export LOCAL_USER_ID ?= $(shell id -u $$USER)
+
+# -------------------
+# --- DEFINITIONS ---
+# -------------------
+
+require-%:
+	@ if [ "${${*}}" = "" ]; then \
+		echo "ERROR: Environment variable not set: \"$*\""; \
+		exit 1; \
+	fi
+
+# -----------------
+# --- MIX TASKS ---
+# -----------------
+
+dev:
+	@MIX_ENV=dev mix ecto.setup && iex -S mix phx.server 
+
+test:
+	@MIX_ENV=test mix test
+
+compile:
+	@mix do deps.get, compile
```

Comment:
Perhaps add ` --warnings-as-errors`?

Reply:
I added it on CI, but not here, b/c we shouldn't treat warnings as errors locally.